src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,32 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
```

Comment:
I'm sure this is just being delayed until closer to merge, but all phpdoc in this PR will need the appropriate `@since` tags.